doc/environ.sh.sample

```diff
@@ -107,6 +107,7 @@ export KOS_CFLAGS=""
 export KOS_CPPFLAGS=""
 export KOS_LDFLAGS=""
 export KOS_AFLAGS=""
+export KOS_SH4_PRECISION=""
```

Comment:
Are you doing this here so that they can comment out the one below and have everything still work or? Because the one below is just overriding this empty default? The others have empty defaults because the flags get appended to these variables rather than having them just get set outright.

Reply:
I went back and forth on whether I should include this line, and my justification was that I thought one of the other ones also didn't add to itself, but looking at the other ones, they all do, so it's probably cleaner to remove it.